project_role/models/project_role.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools.translate import html_translate
+
+
+class ProjectRole(models.Model):
+    _name = 'project.role'
+    _description = 'Project Role'
+
+    name = fields.Char(
+        'Name',
+        translate=True,
+        required=True,
+    )
+    description = fields.Html(
+        string='Description',
+        translate=html_translate,
+    )
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string='Company',
+        required=True,
```

Comment:
This shouldn't be required if you want the same role across several companies, and it has all the sense for multi-company users.

Reply:
You have to add as well a record rule for multi-company restrictions.

Reply:
Re: shouldn't be required - you mean to unlink it from company? What are the common usecases for multi-company setups?

Re: record rule - I'm not 100% sure what exactly you mean, a link to something similar would be much appreciated.

Reply:
Yes, if a role doesn't have any company filled, it's available for all companies. It's the normal use for a lot of objects in Odoo. See any of the multi-company rules for the core objects:

![seleccion_061](https://user-images.githubusercontent.com/7165771/49523078-4aa09700-f8a9-11e8-93d3-42ca8e29998e.png)

Reply:
Oh dear, you force me to learn some dark voodoo SQL magic for that :)))